server/server.js

```diff
@@ -13,18 +13,18 @@ module.exports = function (env) {
   }
 
   // Auth
-  var oauth2 = require('simple-oauth2')({
-    clientID: env.get('GITHUB_CLIENT_ID'),
-    clientSecret: env.get('GITHUB_CLIENT_SECRET'),
-    site: 'https://github.com/login',
-    tokenPath: '/oauth/access_token'
-  });
-  var redirectUri = env.get('AUDIENCE') + '/auth/callback';
-  var authUri = oauth2.AuthCode.authorizeURL({
-    redirect_uri: redirectUri,
-    scope: 'user:email',
-    state: '3(#0/!~'
-  });
+  // var oauth2 = require('simple-oauth2')({
+  //   clientID: env.get('GITHUB_CLIENT_ID'),
+  //   clientSecret: env.get('GITHUB_CLIENT_SECRET'),
+  //   site: 'https://github.com/login',
+  //   tokenPath: '/oauth/access_token'
+  // });
+  // var redirectUri = env.get('AUDIENCE') + '/auth/callback';
+  // var authUri = oauth2.AuthCode.authorizeURL({
+  //   redirect_uri: redirectUri,
+  //   scope: 'user:email',
+  //   state: '3(#0/!~'
+  // });
```

Comment:
No need to comment out, simply remove.
If the need be, we can come back to this commit for re-implementing gh-oauth.